pkg/cli/root.go

```diff
@@ -322,17 +382,9 @@ func LoadConfiguration(config *config.Config, configPath string) {
 		panic(err)
 	}
 
-	// various config checks
-	validateConfiguration(config)
-
 	// defaults
-	defaultTLSVerify := true
+	applyDefaultValues(config, viperInstance)
```

Comment:
Nice hook here.

Reply:
But where is this being used in the server/controller path?

Reply:
For that part LoadConfiguration() is called and applyDefaultValues() inside it.